husarion_ugv_manager/behavior_trees/lights.xml

```diff
@@ -1,84 +1,118 @@
-                                      _onSuccess="charging_anim_percent = battery_percent_round; \
-current_anim_id = CHARGING_BATTERY_ANIM_ID"/>
-          <TickAfterTimeout timeout="20.0"
-                            _skipIf="!e_stop_state">
-            <CallSetLedAnimationService name="SetEStopAnimation"
-                                        id="{E_STOP_ANIM_ID}"
+          <Delay delay_msec="300">
+            <CallSetLedAnimationService name="SetChargerInsertedAnimation"
+                                        id="{CHARGER_INSERTED_ANIM_ID}"
                                         param=""
                                         repeating="false"
+                                        service_name="lights/set_animation"
+                                        _skipIf="current_battery_anim_id == CHARGING_BATTERY_ANIM_ID ||&#10;current_battery_anim_id == BATTERY_CHARGED_ANIM_ID"/>
+          </Delay>
+          <Delay delay_msec="2500">
+            <CallSetLedAnimationService name="SetChargingAnimation"
+                                        id="{CHARGING_BATTERY_ANIM_ID}"
+                                        param="{battery_percent_round}"
+                                        repeating="true"
+                                        service_name="lights/set_animation"
+                                        _skipIf="(battery_percent_round == charging_anim_percent  \&#10;&amp;&amp; current_battery_anim_id == CHARGING_BATTERY_ANIM_ID\&#10;|| battery_percent_round == 1.0)"
+                                        _onSuccess="charging_anim_percent = battery_percent_round; \&#10;current_battery_anim_id = CHARGING_BATTERY_ANIM_ID"/>
+          </Delay>
+          <Delay delay_msec="2500">
+            <CallSetLedAnimationService name="SetChargedAnimation"
+                                        id="{BATTERY_CHARGED_ANIM_ID}"
+                                        param="{battery_percent_round}"
+                                        repeating="true"
+                                        service_name="lights/set_animation"
+                                        _skipIf="battery_percent_round != 1.0 \&#10;|| current_battery_anim_id == BATTERY_CHARGED_ANIM_ID"
+                                        _onSuccess="charging_anim_percent = battery_percent_round; \&#10;current_battery_anim_id = BATTERY_CHARGED_ANIM_ID"/>
+          </Delay>
+        </Parallel>
+      </Sequence>
+      <CallSetLedAnimationService name="SerErrorAnimation"
+                                  id="{ERROR_ANIM_ID}"
+                                  param=""
+                                  repeating="true"
+                                  service_name="lights/set_animation"
+                                  _skipIf="battery_status != POWER_SUPPLY_STATUS_UNKNOWN \&#10;|| current_error_anim_id == ERROR_ANIM_ID"
+                                  _onSuccess="current_error_anim_id = ERROR_ANIM_ID"/>
```

Comment:
What will happen if error is no longer present? Error is at the top level and is repeating, will the animation be cleared if error disappears?